Client/game_sa/CStreamingSA.cpp

```diff
@@ -171,3 +171,11 @@ std::uint32_t CStreamingSA::GetMemoryUsed() const
 {
     return *reinterpret_cast<std::uint32_t*>(0x8E4CB4);
 }
+
+void CStreamingSA::RemoveBigBuildings()
+{
+    using Function_RemoveBigBuildings = void(__cdecl*)();
+    auto removeBigBuildings = reinterpret_cast<Function_RemoveBigBuildings>(0x4093B0);
+
+    removeBigBuildings();
+}
```

Comment:
I think, it's bad class for this method

Reply:
According to https://github.com/gta-reversed/gta-reversed-modern/blob/8416b041a0e96f0df5c9d6915d0ca93ce2dfeaf2/source/game_sa/Streaming.cpp#L2187-L2198 this class should be good.